spec.bs

```diff
@@ -1978,6 +2009,11 @@ An auction config is a [=struct=] with the following items:
 :: Null or an {{unsigned short}}, initially null.
   Optional identifier for an experiment group to support coordinated experiments with buyers'
   trusted servers for buyers without a specified experiment group.
+: <dfn>pending promise count</dfn>
+:: An integer, initially 0. The number of [=auction config/auction signals=],
+  [=auction config/seller signals=], [=auction config/per buyer signals=],
+  [=auction config/direct from seller signals prefix=], or
+  [=auction config/per buyer timeouts=] that are pending {{Promise}}s.

```

Comment:
nit: instead of saying "that are pending Promises" I'd say "whose Promises are not yet resolved"